axelrod/load_data_.py

```diff
@@ -0,0 +1,45 @@
+import pkg_resources
```

Comment:
I was thinking that we could have a `tests/unit/test_load_data.py` file that just checks that these functions run and that the data is of the expected format separately.